tests/end_to_end/test_vulnerability_detector/data/test_cases/cases_vulnerability_detector.yaml

```diff
@@ -0,0 +1,16 @@
+- name: detect_vulnerability
```

Comment:
We should implement two test cases, one for each os.

Reply:
Done in 50b8bfccac4a8d86968c8312a76d406d9f104a98